changelog/add-6707-proxies

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: dev
+
+Adding proxies to src, used to access code outside of it (hooks, globals, functions, and static methods).
```

Comment:
```suggestion
Adding HooksProxy and LegacyProxy to src, used to access code outside of it (hooks, globals, functions, and static methods).
```

Reply:
Fixed in https://github.com/Automattic/woocommerce-payments/pull/6908/commits/c57fdca57b9394a3ea564c475aee875b7b32d4ba.